### ISSUE TITLE
Green Hamburger test, not working with my CSS

### ISSUE THREAD

**Issue by srobinwaters** [freeCodeCamp/testable-projects-fcc#473]
#### Issue Description
it's not compatible with my css, apparently. If I just run it with html only it works fine, but once I paste in my css, the hamburger gets frozen at the top lefthand side of the screen, unresponsive to clicks. If I paste my css in while the hamburger is in the clicked/open state, it seems to run the test fine at first and shows the fraction of the tests that have or have not been passed. But it shows no details as to what tests where failed and how to fix them. 

#### Browser Information
codepen

* Chrome, latest
* OSx latest version
* Macbook Air
#### Your Code / Link to Your Pen
`<p data-height="265" data-theme-id="0" data-slug-hash="QBdxLY" data-default-tab="html,result" data-user="srobinwaters" data-embed-version="2" data-pen-title="Chicken Minion Supply Co" class="codepen">See the Pen <a href="https://codepen.io/srobinwaters/pen/QBdxLY/">Chicken Minion Supply Co</a> by srobinwaters (<a href="https://codepen.io/srobinwaters">@srobinwaters</a>) on <a href="https://codepen.io">CodePen</a>.</p>
<script async src="https://static.codepen.io/assets/embed/ei.js"></script>`
```


 
 
 
```

#### Screenshot

![image](https://user-images.githubusercontent.com/41029754/43038584-e987b8b6-8ce9-11e8-832f-0ae0c3f67f69.png)

**Comment by srobinwaters** [comment]
Worked this problem out with the great folks at the Free Code Camp forum:
[Hamburger Solution](https://forum.freecodecamp.org/t/green-hamburger-test-not-clickable/210623?u=srobinwaters)

**Comment by hbar1st** [comment]
please note this is still an open issue because the css in the script is conflicting with the css that srobinwaters originally was using. (and therefore the script needs to be adjusted to avoid that)

**Comment by srobinwaters** [comment]
The points of conflict that where addressed: 
```
input[type=checkbox]
{
  display: none;
}
```
needed to be changed to use an id instead
```
#show-menu[type=checkbox]
{
  display: none;
}
```
and
```
li ul
{
display: none;
}
```
was changed to use  a class instead
.hidden
{
display: none;
}

**Comment by tchaffee** [comment]
Should be fixed when PR #490 is merged.

**Comment by tbushman** [comment]
Fixed in Chrome and Safari. Closing.
